src/Fantomas.Core.Tests/DotGetTests.fs

```diff
@@ -1245,7 +1190,8 @@ m
     .Property(fun p -> p.Name)
     .IsRequired()
     .HasColumnName("ModelName")
-    .HasMaxLength 64
+    .HasMaxLength
+    64
```

Comment:
@nojaf shouldn't this single line above have one extra level of indentation?

Reply:
No, I don't think so, `64` is the argument of the DotGet application.
Syntactically, this is the equivalent of 
```fsharp
let f =
    m
        .Property(fun p -> p.Name)
        .IsRequired()
        .HasColumnName("ModelName")
        .HasMaxLength

f 64
```

If the application doesn't fit on the rest of the line, the arguments are applied indented on the following line.

```fsharp
f
   64
```

Another extreme example of this would be:
```fsharp
(try
    id
 with ex ->
     id)
    64
```

In practice, I would write the code with parentheses 
```fsharp
    m
        .Property(fun p -> p.Name)
        .IsRequired()
        .HasColumnName("ModelName")
        .HasMaxLength(64)
```

Reply:
I don't know, 64 looks more like the argument that is passed to the last func, and as such, would visually be better place there:

```
    .Property(fun p -> p.Name)
    .IsRequired()
    .HasColumnName("ModelName")
    .HasMaxLength 64
    .HasMaxLength
        64
```

But this is just because that 64 is the last one and is not another fluent call (it doesn't start with a dot). If it started with a dot, then I'd agree with you.

Reply:
```fsharp
m
    .Property(fun p -> p.Name)
    .IsRequired()
    .HasColumnName("ModelName")
    .HasMaxLength 64
    .HasMaxLength
        64
```
This is no longer part of the same chain that starts with `m`.
What you wrote there is:
```fsharp
(m.Property(fun p -> p.Name).IsRequired().HasColumnName("ModelName").HasMaxLength)
     (64.HasMaxLength 64)
```

You can only add that space at the very end, the same reason `"ModelName"` is not applied with a space but has parentheses.

Reply:
I didn't mean to write HasMaxLength twice, that was a typo, I edited my message 5 seconds after posting it